modules/pipeline/providers/definition/get_version_lock.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package definition
+
+import (
+	"fmt"
+
+	"github.com/erda-project/erda-proto-go/core/pipeline/definition/pb"
+	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/modules/pipeline/services/apierrors"
+)
+
+type GetPipelineDefinitionVersionRequestBo struct {
```

Comment:
what is `Bo`

Reply:
![image](https://user-images.githubusercontent.com/28723047/131940218-2f9c5ee1-2b9f-43c2-9690-45c4ca2970e7.png)